sheets-import/README.md

```diff
@@ -1,102 +1,271 @@
+a unique id for each parameter, described below. See [myService.gs](myService.gs)
+for a comprehensive example.
+
+To configure parameters for your service, follow the examples in
+[myService.gs](myService.gs). The options are also documented here.
+
+**Text Field**
+
+<table>
+  <tr><th>Property</th><th>Type</th><th>Description</th><th>Required?</th></tr>
+
+  <tr>
+    <td>id</td>
+    <td>String</td>
+    <td>A unique ID for this param.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>type</td>
+    <td>String</td>
+    <td>Must be "text" for a Text Field.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>label</td>
+    <td>String</td>
+    <td>The human-readable label above the param.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>helper</td>
+    <td>String</td>
+    <td>The human-readable helper text below the label.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>value</td>
+    <td>String</td>
+    <td>The default value of the text field.</td>
+    <td>No</td>
+  </tr>
+</table>
+
+**Checkbox**
+
+<table>
+  <tr><th>Property</th><th>Type</th><th>Description</th><th>Required?</th></tr>
+
+  <tr>
+    <td>id</td>
+    <td>String</td>
+    <td>A unique ID for this param.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>type</td>
+    <td>String</td>
+    <td>Must be "checkbox" for a checkbox.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>label</td>
+    <td>String</td>
+    <td>The human-readable label above the param.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>helper</td>
+    <td>String</td>
+    <td>The human-readable helper text below the label.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>value</td>
+    <td>Bool</td>
+    <td>The default value of the checkbox.</td>
+    <td>No</td>
+  </tr>
+</table>
+
+**Select Dropdown**
+
+<table>
+  <tr><th>Property</th><th>Type</th><th>Description</th><th>Required?</th></tr>
+
+  <tr>
+    <td>id</td>
+    <td>String</td>
+    <td>A unique ID for this param.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>type</td>
+    <td>String</td>
+    <td>Must be "select" for a Select Dropdown.</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>label</td>
+    <td>String</td>
+    <td>The human-readable label above the param.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>helper</td>
+    <td>String</td>
+    <td>The human-readable helper text below the label.</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>options</td>
+    <td>Array</td>
+    <td>Array of options where an option has the format <br/> "{value: "", label: ""}"</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>value</td>
+    <td>String</td>
+    <td>The default value of the select dropdown. Must be one of the `value`s from the `options` array.</td>
+    <td>Yes</td>
+  </tr>
+</table>
+
+## Extending the Template
+
+The **Sheets-Import** template makes use of [AngularJS](https://angularjs.org).
+If you're not familiar with Angular, we recommend working through the
+[tutorial on the site](https://docs.angularjs.org/tutorial).
+
```

Comment:
I don't see ngAppsScript.js.html or app.js.html mentioned on this list; please provide a short description of those as well.